assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -369,6 +369,9 @@ che.infra.kubernetes.workspace_start_timeout_min=8
 # Defines the timeout in minutes that limits the period for which Kubernetes Ingress become ready
 che.infra.kubernetes.ingress_start_timeout_min=5
 
+# Defines the timeout in minutes that limits the max period of result waiting for plugin broker.
+che.infra.kubernetes.plugin_broker_wait_timeout_min=3
```

Comment:
Please rename it in the way in which other plugin broker properties are named, like `che.workspace.plugin_broker.wait_timeout_min`:
https://github.com/eclipse/che/blob/d82481bff1f08383637c84cab1a36d1a65c242bf/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties#L510-L514

And put it near them